maven/src/main/java/org/owasp/dependencycheck/maven/ProxyConfig.java

```diff
@@ -1,17 +1,95 @@
+/*
+ * This file is part of dependency-check-core.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright (c) 2023 Jeremy Long. All Rights Reserved.
+ */
 package org.owasp.dependencycheck.maven;
 
 /**
  * Proxy configuration options.
+ *
+ * @author Jeremy Long
  */
 public class ProxyConfig {
 
-	String host;
-	
-	int port = 8080;
-	
-	/**
-	 * ID od server in Maven settings.xml, <username> and <password> will be used.
-	 */
-	String serverId;
+    /**
+     * The proxy host.
+     */
+    private String host;
+    /**
+     * The proxy port.
+     */
+    private int port = 8080;
+    /**
+     * ID od server in Maven settings.xml, &lt;username&gt; and &lt;password&gt; will be
```

Comment:
```suggestion
     * ID of server in Maven settings.xml, &lt;username&gt; and &lt;password&gt; will be
```
While at it better to also fix this existing typo